community/csv/src/main/java/org/neo4j/csv/reader/Extractors.java

```diff
@@ -952,6 +1008,186 @@ public AnyValue value()
         }
     }
 
+    public static class DateExtractor extends AbstractSingleValueExtractor<AnyValue>
```

Comment:
It looks to me as if PointExtractor and all new Temporal extractors could share the field `value` and the methods `clear()` and `value()`. Maybe create an abstract type `AbstractSingleAnyValueExtractor extends AbstractSingleValueExtractor<AnyValue>` ?